test/data/samples/fwjr_production.py

```diff
@@ -116,16 +47,16 @@
                          'globalTag': 'GR_R_62_V3::All',
                          'guid': 'ECCFE421-08CB-E511-9F4C-02163E017804',
                          'inputDataset': '/Cosmics/Run2011A-v1/RAW',
-                         'inputLFNs': ['/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/E8099605-8853-E011-A848-0030487A18F2.root'],
-                         'inputPFNs': ['root://eoscms.cern.ch//eos/cms/store/data/Run2011A/Cosmics/RAW/v1/000/160/960/E8099605-8853-E011-A848-0030487A18F2.root'],
+                         'inputLFNs': [0],
+                         'inputPFNs': [1],
                          #TODO change to empty string from None
                          'location': '',
                          'merged': False,
                          'module_label': 'ALCARECOStreamDtCalib',
                          'ouput_module_class': 'PoolOutputModule',
                          'outputDataset': '/Cosmics/CMSSW_7_0_0_pre11-DtCalib-RECOCOSD_TaskChain_Data_pile_up_test-v1/ALCARECO',
-                         'outputLFN': '/store/unmerged/CMSSW_7_0_0_pre11/Cosmics/ALCARECO/DtCalib-RECOCOSD_TaskChain_Data_pile_up_test-v1/00000/ECCFE421-08CB-E511-9F4C-02163E017804.root',
-                         'outputPFN': 'root://eoscms.cern.ch//eos/cms/store/unmerged/CMSSW_7_0_0_pre11/Cosmics/ALCARECO/DtCalib-RECOCOSD_TaskChain_Data_pile_up_test-v1/00000/ECCFE421-08CB-E511-9F4C-02163E017804.root',
+                         'outputLFN': 2,
```

Comment:
Seangchan, here you assign outputLFN to be int, while few lines above infputLFNs are type list(int). I want to double check if outputLFN will be singles? May be for safety, generalization and consistency we need to assign list(int) in both cases.